app/templates/components/forms/admin/settings/ticket-fees-form.hbs

```diff
@@ -0,0 +1,41 @@
+<form class="ui form" {{action 'submit' on='submit'}}>
+  <h3 class="ui header">
+    {{t 'List of supported currencies'}}
+  </h3>
+  {{#each currencies as |currency index|}}
+    <div class="ui sub header">
+      {{currency.code}} - {{currency.name}} ({{currency.symbol}})
+    </div>
+    <div class="field">
+      <div class="label">
+        {{t 'Service Fee'}}
+      </div>
+      <div class="ui right labeled input">
+        {{input type='number' name=(concat currency.code '_service') value=(get (object-at index fee) 'serviceFee') step=0.1 min=0}}
```

Comment:
You're assuming the passed in fee is in the same order as currencies. It need not be.